client/lib/js/react/components/accounts.js

```diff
@@ -52,10 +54,10 @@ const Accounts = React.createClass({
                         })
                         .map(account => (
                             <AccountRow
-                                key={account.name}
                                 account={account}
                                 activeAccount={activeAccount}
-                                handleClick={handleAccountClick} />
+                                handleClick={handleAccountClick}
+                                key={account.name} />
```

Comment:
Actually don't bother with these lint warnings; I think the `key` (and `ref`) props should always go first since they're special to React, but I haven't gotten around to extending the rule to allow for this.